pdns/qtype.hh

```diff
@@ -38,206 +36,123 @@
 
 */
 
-
-
 class QType
 {
 public:
-  QType(); //!< Naked constructor
-  explicit QType(uint16_t); //!< convert from an integer to a QType
-  QType(const QType& orig) : code(orig.code)
+  QType(uint16_t qtype = 0) : code(qtype) {}
```

Comment:
I'm not sure it's such a good idea to remove the `explicit` keyword on the constructor from `uint16_t`?

Reply:
It's conventient to be able to pass e.g. `QType::A` to a function taking a `QType`.

Reply:
That makes sense, so let's keep it that way.